runserver.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
 
-pip install -r requirements.txt
```

Comment:
This was a development hack to make sure all my pip packages were installed, shouldn't have been pushed in the first place.